vars/debian-13.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+# Override variables for Debian 13 
+
+openwisp2_ntp_pkg: ntpsec
+openwisp2_libgdk_pixbuf_pkg: libgdk-pixbuf-2.0-0
+freeradius_dir: /etc/freeradius/3.0
```

Comment:
Debian 13 has FreeRADIUS 3.7 available in Debian feeds. It's config is present in `/etc/freeradius/3.0/` directory instead of the default `/etct/freeradius/`